src/Microsoft.Health.Fhir.TemplateManagement/Models/ImageInfo.cs

```diff
@@ -50,12 +53,17 @@ public string Label
 
         public bool IsDefaultTemplate()
         {
-            return string.Equals(ImageReference, DefaultTemplateImageReference, StringComparison.InvariantCultureIgnoreCase);
+            return IsDefaultTemplateImageReference(ImageReference);
+        }
+
+        public static string GetDefaultTemplateImageReferenceByDatatype(DataType datatype)
+        {
+            return DefaultTemplateInfo.DefaultTemplateMap.GetValueOrDefault(datatype).ImageReference;
         }
 
         public static bool IsDefaultTemplateImageReference(string imageReference)
         {
-            return string.Equals(imageReference, DefaultTemplateImageReference, StringComparison.InvariantCultureIgnoreCase);
+            return string.Equals(imageReference, DefaultTemplateImageReference, StringComparison.InvariantCulture) || DefaultTemplateInfo.DefaultTemplateMap.Values.Any(value => string.Equals(imageReference, value.ImageReference, StringComparison.InvariantCulture));
```

Comment:
Is case sensitive?

Reply:
Yes, it is case-sensitive while getting default templates, so I just modify the code here to keep consistent.

Reply:
@moria97 May be ignore case is more reasonable?

Reply:
Per our offline discussion, default templates are case insensitive and customers's acr is case sensitive.